doc/examples/_valid_examples.toml

```diff
@@ -111,6 +111,7 @@ files = [
     "reconst_qti.py",
     "reconst_qtiplus.py",
     "histology_resdnn.py",
+    "bingham_fit_fodf.py"
```

Comment:
I forgot to add the new tutorial here - we have to do that.
Also I didn't delete the old example "bingham_fit_fodf.py". What should we do with it? Integrated it with the new tutorial? Delete it? This is something that we need to discuss.